tests/azure/templates/galaxy_pytest_script.yml

```diff
@@ -0,0 +1,61 @@
+---
+parameters:
+  - name: build_number
+    type: string
+  - name: scenario
+    type: string
+    default: fedora-latest
+  - name: ansible_version
+    type: string
+    default: ""
+  - name: python_version
+    type: string
+    default: 3.x
+
+jobs:
+- job: Test_PyTests
+  displayName: Run pytests on ${{ parameters.scenario }}
+  timeoutInMinutes: 120
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '${{ parameters.python_version }}'
+
+  - script: |
+      pip install \
+        "molecule[docker]>=3" \
+        "ansible${{ parameters.ansible_version }}"
+    displayName: Install molecule and Ansible
+
+  - script: ansible-galaxy collection install community.docker ansible.posix
+    displayName: Install Ansible collections
+
+  - script: pip install -r requirements-tests.txt
+    displayName: Install dependencies
+  - script: |
+      utils/build-galaxy-release.sh
+      ansible-galaxy collection install freeipa-ansible_freeipa-$(git describe --tags 2>/dev/null | sed -e "s/^v//").tar.gz -vvv
```

Comment:
As this is a fresh checkout, there should not be any other collection for galaxy. It might be possible to simply use `ansible-galaxy collection install freeipa-ansible_freeipa-*.tar.gz` here.

What do you think about adding a "--install" to `build-galaxy-release.sh` to automatically install the generated version? This way it would not be needed to get the proper tag for the built collection.

Reply:
Thank you for adding the '-i' option to `build-galaxy-release.sh`. I changed the script to use the option.